lib/fs.js

```diff
@@ -2283,11 +2456,19 @@ WriteStream.prototype.destroySoon = WriteStream.prototype.end;
 
 // SyncWriteStream is internal. DO NOT USE.
 // This undocumented API was never intended to be made public.
-var SyncWriteStream = internalFS.SyncWriteStream;
+var SyncWriteStream = _SyncWriteStream;
 Object.defineProperty(fs, 'SyncWriteStream', {
   configurable: true,
   get: internalUtil.deprecate(() => SyncWriteStream,
                               'fs.SyncWriteStream is deprecated.', 'DEP0061'),
   set: internalUtil.deprecate((val) => { SyncWriteStream = val; },
                               'fs.SyncWriteStream is deprecated.', 'DEP0061')
 });
+
+Object.defineProperty(fs, 'async', {
+  enumerable: true,
+  configurable: true,
+  get() {
+    return require('internal/async/fs');
+  }
```

Comment:
Why not `value` + `writable: false`?

Reply:
to avoid the experimental warning from printing until someone explicitly goes to use the new API. We can switch this later once it's non-experimental